e2e-tests/tests/features/multinamespace_psmdb/10-assert.yaml

```diff
@@ -70,11 +71,3 @@ spec:
           endpointUrl: s3.amazonaws.com
           region: us-east-2
         type: s3
-status:
-  replsets:
-    rs0:
-      initialized: true
-      ready: 1
-      size: 1
-      status: ready
-  state: ready
```

Comment:
Well, I removed it because it was always timing out and we don't check it for the other DBEngines.
Giving it a second, I think we should though.
I'll add this when I add the VMAgent tests in a follow-up PR.
Thank you for pointing it out.